src/Mailbox.php

```diff
@@ -134,6 +135,24 @@ public function getStatus(int $flags = null): \stdClass
         return $return;
     }
 
+    /**
+     * Get Mailbox quota.
+     *
+     * @param string $root
+     *
+     * @return array
+     */
+    public function getQuota(string $root = 'INBOX'): array
+    {
+        $return = \imap_get_quotaroot($this->resource->getStream(), $root);
+
+        if (false === $return) {
```

Comment:
This is not enough: if the client doesn't support the feature, a warning will raise, but this library aims to strictness to we need to throw an Exception, take a look at:

https://github.com/ddeboer/imap/blob/8d89b248800eed01c34764b3c6d1880a743e6540/src/Server.php#L90-L106

Reply:
OK I understand this case.